packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -26,9 +30,9 @@ describe('Approve token for deposit', () => {
 
       cy.findByText('MAX').click()
 
-      cy.findGasFeeSummary(zeroToLessThanOneETH)
-      cy.findGasFeeForChain(getL1NetworkName(), zeroToLessThanOneETH)
-      cy.findGasFeeForChain(getL2NetworkName(), zeroToLessThanOneETH)
+      cy.findGasFeeSummary(zeroToLessThanOneEth)
```

Comment:
In case of custom native token chain, the gas summary contains ETH + Native Token but we're only checking for ETH component here. Although the next 2 lines check for the fas fee per chain individually, just flagging it here.

<img width="550" alt="image" src="https://github.com/user-attachments/assets/000a50ac-e4f9-4a49-a5d0-e82210ef8cb0">

Reply:
And we're checking `findGasFeeSummary` in multiple files.